frontend/src/index.html

```diff
@@ -200,6 +202,11 @@ <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{#helpers.hig
                     </a>
                   </div>
                 </div>
+                <div x-show="is_open" @click="text_open=false" class="mt-1">
```

Comment:
- Don't close the topics on click. That will not be convenient. People would need to select and copy some text from it.
- Do we really need a wrapper `div` here? Can't we apply directives and classes directly to the nested element?